libs/data-service-generator/src/admin/gitignore/create-gitignore.ts

```diff
@@ -0,0 +1,31 @@
+import {
+  CreateAdminGitIgnoreParams,
+  EventNames,
+  Module,
+} from "@amplication/code-gen-types";
+import DsgContext from "../../dsg-context";
+import { readCode } from "@amplication/code-gen-utils";
+import pluginWrapper from "../../plugin-wrapper";
+
+const templatePath = require.resolve("./template.gitignore");
+
+export function createGitIgnore(
+  eventParams: CreateAdminGitIgnoreParams = {}
+): Module[] {
+  return pluginWrapper(
+    createGitIgnoreInternal,
+    EventNames.CreateAdminGitIgnore,
+    eventParams
+  );
+}
```

Comment:
I think we should export the content of the file as a json object and allow to manipulate the file in the event
I would expect to see a single CreateGitIgnore for both the server and the client, with the same logic and a different data object for initialization